snapcast-control/Dockerfile.template

```diff
@@ -0,0 +1,26 @@
+FROM balenalib/%%BALENA_MACHINE_NAME%%-debian:buster
```

Comment:
Maybe we should try multi-stage dockerfile or base it on alpine to try and reduce image size.